postgres-appliance/configure_spilo.py

```diff
@@ -229,49 +257,45 @@ def get_dcs_config(config, placeholders):
     return config
 
 
-def write_wale_command_environment(placeholders):
+def write_wale_command_environment(placeholders, overwrite):
     az = get_instance_meta_data('placement/availability-zone') or 'dummy-region'
 
     if not os.path.exists(placeholders['WALE_ENV_DIR']):
         os.makedirs(placeholders['WALE_ENV_DIR'])
 
-    with open(os.path.join(placeholders['WALE_ENV_DIR'], 'WALE_S3_PREFIX'), 'w') as f:
-        f.write('s3://{WAL_S3_BUCKET}/spilo/{SCOPE}/wal/'.format(**placeholders))
+    write_file('s3://{WAL_S3_BUCKET}/spilo/{SCOPE}/wal/'.format(**placeholders),
+               os.path.join(placeholders['WALE_ENV_DIR'], 'WALE_S3_PREFIX'), overwrite)
+    write_file('https+path://s3-{}.amazonaws.com:443'.format(az[:-1]),
+               os.path.join(placeholders['WALE_ENV_DIR'], 'WALE_S3_ENDPOINT'), overwrite)
+
 
-    with open(os.path.join(placeholders['WALE_ENV_DIR'], 'WALE_S3_ENDPOINT'), 'w') as f:
-        f.write('https+path://s3-{}.amazonaws.com:443'.format(az[:-1]))
+def write_crontab(placeholders, path, overwrite):
 
+    if not overwrite:
+        with open(os.devnull, 'w') as devnull:
+            cron_exit = subprocess.call(['sudo', '-u', 'postgres', '-l'], stdout=devnull, stderr=devnull)
```

Comment:
sudo -u postgres -l doesn't have anything to do with the crontab. Did you intend to write sudo -u postres crontab -l?